client/src/Stats.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { Link, useRouteMatch } from 'react-router-dom'
+import placeholder from './images/Placeholder.png'
+import star from './images/star.svg'
+import back from './images/left-chevron.svg'
+import gameList from './gameList.json'
+
+export default function Stats() {
+  function loadFromStorage(name) {
+    try {
+      return JSON.parse(localStorage.getItem(name))
+    } catch (error) {
+      console.log(error.message)
+    }
+  }
+
+  const match = useRouteMatch()
+
+  return (
+    <ContentWrapper>
+      <div className="navigate">
+        <Link to="/">
+          <img className="navigate__back" src={back} alt="go back"></img>
+        </Link>
+        <h2>Stats</h2>
+        <img className="navigate__bookmark" src={star} alt="bookmark"></img>
+      </div>
+      {gameList.map(
+        (game) =>
+          game.id === parseInt(match.params.gameId) && (
+            <section>
+              <img
+                className="placeholder"
+                src={placeholder}
+                alt="placeholder"
+              ></img>
+              <div className="description">
+                <h3>Title:</h3>
+                <p key={game.id}>{game.title}</p>
+                <h3>Platform:</h3>
+                <p>{game.platform}</p>
+              </div>
+              <div className="info">
+                <h3>Genre:</h3>
+                <p>{game.genre}</p>
+                <h3>Mode:</h3>
+                <p>{game.mode}</p>
+              </div>
+            </section>
+          )
+      )}
+    </ContentWrapper>
+  )
+}
+
+const ContentWrapper = styled.main`
```

Comment:
macht es Sinn main in einer globalStyles zu stylen, damit es sich nicht wiederholt?